reflex/state.py

```diff
@@ -144,6 +144,9 @@
     ValueError,
 )
 
+# For BaseState.get_var_value
+VAR_TYPE = TypeVar("VAR_TYPE", bound=Var)
```

Comment:
You don't actually want the bound here.

Reply:
Fixed, not sure how that wound up there. Been a long week 😅